package.json

```diff
@@ -3,13 +3,8 @@
   "name": "philips-jointspace",
   "version": "0.5.1",
   "dependencies": {
-    "crypto": "^1.0.1",
```

Comment:
The `crypto` dependency is still used during pairing but no longer referenced in the `package.json`.

Reply:
Thanks for clearing things up!